cli/parser/parser.go

```diff
@@ -546,11 +493,7 @@ func getCommandLineSchema(args []string, bazelHelp BazelHelpFunc, onlyStartupOpt
 	if commandOptions, ok := optionSets[schema.Command]; ok {
 		schema.CommandOptions = commandOptions
 	} else {
-		commandHelp, err := bazelHelp(schema.Command)
-		if err != nil {
-			return nil, err
-		}
-		schema.CommandOptions = parseBazelHelp(commandHelp, schema.Command)
+		return nil, fmt.Errorf("Flags proto did not contain options for command '%s'.", schema.Command)
```

Comment:
Should these messages be lowercased?

Reply:
yep, looks like; I just hadn't even noticed. Thanks for the catch!